### PR TITLE
10 - Made header in index.html

### LINKED ISSUE #10: [3] index.html: Header
## Details
Links to (placeholder path): Öppettider & Kontakt | Om Spegeln | Biljettinfo | Vill du ha vårt nyhetsbrev? | Köp presentkort
Logo (link to index.html on click)
Social Media icons (placeholder path links)

Styling SCSS - header.scss
Some hover effect
Design up to the developer, artistic freedom!

Two designs: mobile menu, desktop

## Dependencies
 - variables.scss
 - Logo design

## User story
 As a visitor on the site, I want find out more practical information about the cinema; for instance tickets and opening hours.

## Definition of done
 A nice looking menu (with styling!), both on desktop and mobile. Hover effects.

## Tests
 Mouseover a link to see hover effects.

### PR DESCRIPTION
**Pull request by ved0**
Created header in index.html both for mobile and desktop. 
*added a button in index.html
*created header.scss
*added images for social media

### CODE REVIEW

**Comment by Arash-SJPI22** [comment]
Looks good, maybe use .svg files for logos like that for better scalability and often smaller file size :)

File: index.html [hunk 1]
```diff
@@ -1,148 +1,168 @@
 <!DOCTYPE html>
 <html lang="sv">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- cgabge tthis -->
```
Comment (saeraa) [inline index.html]: cgabge?

File: js/hamburgerMenu.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const toggleButton = document.querySelector(".header-list-toggle");
+const headerList = document.querySelector(".header-list");
+
+toggleButton.addEventListener("click",()=>{
+    toggleButton.classList.toggle("open");
+    if(headerList.classList.toggle("open")){
+        document.body.style.overflowY = "hidden";
+    } else {
+        document.body.style.overflowY = "auto";
+    }
```
Comment (saeraa) [inline js/hamburgerMenu.js:10]: Neat solution!

File: scss/header.scss [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    }
+  }
+}
+
+@media (min-width: $bp-small) and (max-width: 1000px) {
+  header {
+    width: 90%;
+    margin-top: 2em;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (saeraa) [inline scss/header.scss:107]: For future reference, these two lines could be replaced by:
`margin-inline: auto;`

**Review by saeraa — APPROVED** [review]
Looking good. As Arash also mentioned, the images should probably be svgs for scalability and size - but it doesn't matter a whole lot in this project. 😁

File: scss/header.scss (hunk 3 above)
Comment (ved0) [reply scss/header.scss:107]: Nice tip! Thanks :)

File: js/hamburgerMenu.js (hunk 2 above)
Comment (ved0) [reply js/hamburgerMenu.js:10]: Thank you! 👍

File: index.html (hunk 1 above)
Comment (ved0) [reply index.html]: Haha, wops... And I thought I wrote change this even.. removed it now

**Comment by ved0** [comment]
> Looks good, maybe use .svg files for logos like that for better scalability and often smaller file size :)

really good advice. Thank you! 💯

**Comment by ved0** [comment]
> Looking good. As Arash also mentioned, the images should probably be svgs for scalability and size - but it doesn't matter a whole lot in this project. 😁

Thanks, I appreciate it and its definitely good for future practice.

**Pull request merged**